package.json

```diff
@@ -65,7 +65,7 @@
     "prettier-plugin-organize-imports": "^3.2.4",
     "prettier-plugin-solidity": "^1.3.1",
     "puml-for-markdown": "^1.2.1",
-    "sol2uml": "^2.5.20",
+    "sol2uml": "^2.5.19",
```

Comment:
Is it to align with voucher project or you had issues with the old version?

Reply:
I ran into an issue with .20 yes, so back to .19, which is aligned with voucher.
Last time I forgot to follow my own comment when bumping..!
https://github.com/iExecBlockchainComputing/PoCo/blob/c8ddecf6e69f7735bb4ba68881f620c76e9c8e65/package.json#L75